scripts/release/promote/prep_binary_for_pypi.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Preps binaries for publishing to pypi by removing the
+# version suffix we normally add for all binaries
+# (outside of default ones, CUDA 10.2 currently)
+
+# Usage is:
+# $ prep_binary_for_pypy.sh <path_to_whl_file> <path_to_multiple_whl_files>
+
+# Will output a whl in your current directory
```

Comment:
When specified with multiple whl files, I guess this script should output multiple wheels in the current directory?

Reply:
Yes that's the idea